src/main/lib/file-utils.js

```diff
@@ -41,4 +41,23 @@ const listFilesInDirectory = (dir, excludes = []) => {
     );
 };
 
-module.exports = { writeJson, readJson, listFilesInDirectory, exists };
+function listFilesInDirectoryRec (root, dir, excludes = []) {
```

Comment:
Name does not communicate what it does. A suffix `Rec` is mysterious. Please rename to clearly indicate it travers subdirectories to list all the files.